### ISSUE TITLE
Upload a folder

### ISSUE THREAD

**Issue by mcdoubleyou** [hartez/PneumaticTube#13]
Hey,

is it possible to upload a complete folder instead of only one file?
I try with \* but than the program crashes.

Thx mcdy

**Comment by hartez** [comment]
It's not currently possible, but that's a good idea. I will try to get that into the next version.

In the meantime, if you're using PowerShell you can do something like this:
`dir .\folder | % {PneumaticTube.exe -f $_.FullName -p folder}`

This will run pneumatictube in a loop, uploading each file in the folder one at a time.

If you're at the regular old DOS prompt, something like this should work:
`for %a in ("C:\folder\*") do pneumatictube.exe -f %~fa -p testfolder`

**Comment by mcdoubleyou** [comment]
thank you for feedback and sugestion, in corporation with administrator.de I optimized it.
```
set whatever_dir=C:\temp\whatever
set PneumaticTube=C:\portApps\PneumaticTube\PneumaticTube.exe

for /r %whatever_dir% %%a in ("*") do (
    set "FullPath=%%~dpa"
    set "RelPath=!FullPath:%whatever_dir%=!"
    %PneumaticTube% -f "%%~fa" -p !RelPath!
)
```

of course you have du individualice it

Greetz mcdy

**Comment by hartez** [comment]
Implemented in version 1.2
